src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -170,6 +170,8 @@
     <Compile Include="System\Resources\ResourceSet.cs" />
     <Compile Include="System\Runtime\InteropServices\CriticalHandle.cs" />
     <Compile Include="System\Text\CodePageDataItem.cs" />
+    <Compile Include="System\Threading\ClrThreadPool.Unix.cs" />
+    <Compile Include="System\Threading\ClrThreadPool.HillClimbing.Unix.cs" />
```

Comment:
In this PR there isn't anything Unix-specific; however, in future PRs the rest of the thread pool implementation will have Unix-specific portions. The CPU utilization calculation specifically is the Unix-specific code that Hill climbing will touch.

I can change it if you would like me to do so.